api/src/users/users.service.ts

```diff
@@ -8,13 +8,16 @@ import {
   UnprocessableEntityException,
 } from '@nestjs/common';
 import { InjectModel } from '@nestjs/sequelize';
-import { Optional } from 'sequelize';
+import { Op, Optional } from 'sequelize';
 import { Users } from 'src/db/model/users.model';
 import { UsersDto } from './dto/users.dto';
 import { hash } from 'bcrypt';
 import { UserApplications } from 'src/db/model/user-applications.model';
 import { compare } from 'bcrypt';
 import { Applications } from 'src/db/model/applications.model';
+import * as speakeasy from 'speakeasy';
```

Comment:
Add package.json file to install this library

Reply:
added package.json file